specification/mediaservices/resource-manager/Microsoft.Media/Streaming/stable/2022-11-01/streamingservice.json

```diff
@@ -486,6 +486,153 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Media/mediaservices/{accountName}/liveEvents/{liveEventName}/status": {
+      "get": {
```

Comment:
why is this a get? shouldn't this be a POST?

Reply:
this is to get some status data of one live event. btw, why should it be a POST?